static/js/click_handlers.js

```diff
@@ -547,23 +548,46 @@ export function initialize() {
             onHidden: (instance) => {
                 instance.destroy();
             },
+            onShow: (instance) => {
+                // For both buddy list and top left corner pm list, `target_node`
+                // is their parent `ul` element. We canot use MutationObserver
+                // directly on the reference elmenet because it will be removed
+                // and we need to attach it on an elment which will remain in the
+                // DOM which is their parent `ul`.
+                const target_node = instance.reference.parentElement.parentElement;
```

Comment:
Should we search for the `ul` element here instead? This is faster compared to it though.

Reply:
I don't think speed at that level is important when we're talking about code that only runs in response to user interaction (it'd be very different if it was code that ran in the background).

Reply:
Okay.